packages/client/components/TeamPromptMeeting.tsx

```diff
@@ -190,7 +179,7 @@ const TeamPromptMeeting = (props: Props) => {
                 </ResponsesGridContainer>
               </ErrorBoundary>
             </StyledMeetingHeaderAndPhase>
-            <TeamPromptDiscussionDrawer meetingRef={meeting} isDesktop={isDesktop} />
+            <TeamPromptDrawer meetingRef={meeting} isDesktop={isDesktop} />
```

Comment:
+1 we could do the conditional rendering here instead of doing it internally in `TeamPromptDrawer` and treat `TeamPromptDrawer` as a style container

Reply:
I still prefer to keep everything abstracted within `TeamPromptDrawer`, especially since the internal drawer components need the `onToggleDrawer` handler managed by the `TeamPromptDrawer`